resources/quality/dagoma/dagoma_neva_pla_standard.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 3
+name = Standard
+definition = dagoma_neva
+
+[metadata]
+setting_version = 4
+type = quality
+quality_type = normal
+weight = -1
+material = generic_pla
+
+[values]
+layer_height = 0.15
+
+material_print_temperature = =default_material_print_temperature + 5
+material_bed_temperature_layer_0 = =default_material_print_temperature + 5
```

Comment:
This is probably a mistake, the bed temperature must not depend on the print temperature. It should be `default_material_bed_temperature`

Reply:
Fixed in f1efaa7